doc/source/create-k8s-cluster.rst

```diff
@@ -38,22 +38,33 @@ connect your credit card or other payment method to your google cloud account.
 
 2. Enable the `Kubernetes Engine API <https://console.cloud.google.com/apis/api/container.googleapis.com/overview>`_.
 
-3. Install and initialize the **gcloud command-line tools**. These tools send
-   commands to Google Cloud and lets you do things like create and delete
-   clusters.
+3. Prepare your command line interface. You have two options, using Google's
```

Comment:
3. Use your preferred command line interface.

You have two options: a) use the Google Cloud Shell (no installation needed) or b) install and use the gcloud command-line tool.  If you are unsure which to choose, we recommend beginning with option "a" and use Google Cloud Shell.